debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -59,6 +59,8 @@ CompiledMethodWrapper.dataLength = required
 CompiledMethodWrapper.romMethodOffset = required
 GC_ArrayletObjectModelBase._arrayletRangeBase = required
 GC_ArrayletObjectModelBase._arrayletRangeTop = required
+GC_ArrayletObjectModelBase._enableDoubleMapping = bool
+GC_ArrayletObjectModelBase._enableVirtualLargeObjectHeap = bool
```

Comment:
do we need (or even if are allowed) to do this?
and if it's needed, should it be 'required'?

Reply:
if DDR changes goes before runtime changes, we have to introduce _enableVirtualLargeObjectHeap in runtime (and make it really visibale if under a build flag)